### ISSUE TITLE
403 Access Forbidden by CSRFProtector: when uploading files

### ISSUE THREAD

**Issue by elieobeid7** [mebjas/CSRF-Protector-PHP#94]
My webserver folder is `public_html` . The structure is the following:

- index.php  - holds the html, redirects to controller.php
- controller.php - handles the post request and take care of the uploads then redirects to index.php
- pictures folder, chmod 755
- CSRFProtector is included in all files, I use the default token `csrfp_token`

if I fill the form without uploading, everything is fine. If I upload a picture, I get

`403 Access Forbidden by CSRFProtector! `

**Comment by mebjas** [comment]
This error is coming up at `conroller.php` is it?

**Comment by elieobeid7** [comment]
That's true, only when uploading files, so if no files are selected, everything is fine.

**Comment by elieobeid7** [comment]
closing this due to it being so old, I havent used that library since
